extensions/typescript-language-features/src/typescriptServiceClient.ts

```diff
@@ -558,6 +558,8 @@ export default class TypeScriptServiceClient extends Disposable implements IType
 				providePrefixAndSuffixTextForRename: true,
 				allowRenameOfImportPath: true,
 				includePackageJsonAutoImports: this._configuration.includePackageJsonAutoImports,
+				// @ts-expect-error until TS 5.3
```

Comment:
You can actually remove the `@ts-expect-error` check. We now build using the latest TS 5.3 types